docker_services_cli/docker-services.yml

```diff
@@ -47,6 +47,25 @@ services:
     ports:
       - "9200:9200"
       - "9300:9300"
+  opensearch:
+    image: opensearchproject/opensearch:${OPENSEARCH_VERSION}
+    restart: "unless-stopped"
+    environment:
+      - bootstrap.memory_lock=true
+      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
+      - discovery.type=single-node
+      - plugins.security.audit.type=debug
```

Comment:
We might want to completely [disable security](https://opensearch.org/docs/latest/security-plugin/configuration/disable) in ``docker-services-cli`` since it's used for testing - or at least during migration to make migration from ES easier.